tests/test_api.py

```diff
@@ -296,6 +299,63 @@ def test_url_protocol_association(delete_files):
     )
 
 
+@pytest.mark.skipif(PLATFORM != "win", reason="Windows only")
+def test_windows_terminal_profiles(tmp_path):
+    def _parse_terminal_profiles(settings_file: Path) -> dict:
+        settings = json.loads(settings_file.read_text())
+        return {
+            profile.get("name", ""): profile.get("commandline", "")
+            for profile in settings["profiles"]["list"]
+        }
+
+    settings_files = [
+        # Stable
+        Path(
+            folder_path("user", False, "localappdata"),
+            "Packages",
+            "Microsoft.WindowsTerminal_8wekyb3d8bbwe",
+            "LocalState",
+            "settings.json",
+        ),
+        # Preview
+        Path(
+            folder_path("user", False, "localappdata"),
+            "Packages",
+            "Microsoft.WindowsTerminalPreview_8wekyb3d8bbwe",
+            "LocalState",
+            "settings.json",
+        ),
+        # Unpackaged (Scoop, Chocolatey, etc.)
+        Path(
+            folder_path("user", False, "localappdata"),
+            "Microsoft",
+            "Windows Terminal",
+            "settings.json",
+        ),
+    ]
```

Comment:
These are duplicated in `menunist.platforms.win`. I wonder if we should put them in a function that takes `mode` and returns the same list so we can test it here. Thoughts?

Reply:
I moved it into its own function. I also decided to harden the function in case that seemingly auto-generated string ever gets changed.